### PR TITLE
DOC: update README and gitignore

### PR DESCRIPTION
**Pull request by shdunning**
2 minor changes:
1. add small blurb to README about using pyenv to manage multiple py versions
2. add vscode dir to gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ pyvenv.cfg
 tags
 .coverage
 .pytest_cache/
+.vscode/
```
Comment (Glutexo) [inline .gitignore]: I wouldn’t add local editor folders to the project’s _.gitignore_. They’re not relevant to the project and to the developers who use different editors. These are better be added to a global _.gitignore_ on your machine.

Comment (shdunning) [reply .gitignore]: 👍

**Comment by Glutexo** [comment]
@shdunning Please revert the [_.gitignore_](https://github.com/shdunning/insights-host-inventory/blob/e8acdd230dc4387fa6b585ea71a3380c66ec780d/.gitignore) changes and I‘ll merge.

**Comment by shdunning** [comment]
@Glutexo sorry for the delay - done!

**Pull request merged**